### ISSUE TITLE
Install is broken and just deleted an entire directory of mine

### ISSUE THREAD

**Issue by Lawouach** [pulumi/pulumi#1975]
Hey,

I wanted to install Pulumi today. I grabbed the Linux tarball to do so. Here is the structure I have:

``` 
$ pwd
/home/sylvain/apps
$ tar zxvf pulumi-v0.15.2-linux-x64.tar.gz
$ export PULUMI_INSTALL_PATH=/home/sylvain/apps
$ ./pulumi/install.sh
Installing Pulumi to /home/sylvain/apps
./pulumi/install.sh: ligne 43: /home/sylvain/apps/pulumi/install.log: Aucun fichier ou dossier de ce type
error: There was a problem during installation.  Please join our #community-discussion
    Slack channel (https://pulumi.slack.com/app_redirect?channel=community-discussion) for help.
    We may need the install log file which can be found at:
        /home/sylvain/apps/pulumi/install.log
$ ls | wc -l
0
```

So, that's brilliant, I just lost the entire content of my apps directory where I just wanted to install pulumi.

Whether or not I should have installed from the same directory I wanted to copy the bin to should not have led to that.

- Sylvain

**Comment by ellismg** [comment]
Hi Sylvain,

Let me start of by offering you my deepest apologies for causing you to lose any of your files. Blowing away existing folder is 100% unacceptable, especially in an installer.

I've opened https://github.com/pulumi/pulumi/pull/1992 to remove `install.sh` from the tarball completely, since we no longer really have an install process that needs to be followed.  The comments in the commit in #1992 give some more details, but the long and short of it is that at one time Pulumi needed to be installed into its own directory and installation had to do some system wide things. Neither of these are true today and unfortunately our focus on pushing people towards installing via `curl`'ing from https://get.pulumi.com meant that we never revisited if we actually needed this install script.

Not realizing the script was no longer needed and removing it sooner was 100% my fault.

After the PR is merged, We'll do another release and update alternative instructions on  https://pulumi.io/quickstart/install.html to tell folks to either just put the `pulumi` folder on their `$PATH` or  just copy all the binaries from the `pulumi` directory into an existing directory already on `$PATH`.  If there's anything else you think we should be doing here to make this process smoother (without having to resort to shell scripts that try to be clever), I'd love your feedback.

Again, I am very sorry that you ran into this.  I feel absolutely awful that our mistake caused you to lose files.

**Comment by joeduffy** [comment]
Merged #1992 just now. Please also allow me to apologize as well as thank you for reporting the issue in such a kind and understanding manner.

**Comment by Lawouach** [comment]
Hey folks,

Thank you very much for you kind feedback here. I think there was not too much harm done in the end, while my apps directory was filled with binaries that I needed, it didn't contain anything I can't recover.

It just means I spent some time downloading CLI tools again this week. Clean up is sometimes benefitial.

Anyway, thanks for the fix. I'll be resuming my trial of Pulumi now.

- Sylvain
